src/cdk/schematics/utils/build-component.ts

```diff
@@ -195,6 +201,14 @@ export function buildComponent(options: ComponentOptions,
     validateName(options.name);
     validateHtmlSelector(options.selector!);
 
+    // In case the specified style extension is not part of the supported CSS supersets,
+    // we generate the stylesheets with the "css" extension. This ensures that we don't
+    // accidentally generate invalid stylesheets (e.g. drag-drop-comp.styl) which will
+    // break the Angular CLI project. See: https://github.com/angular/material2/issues/15164
+    if (options.styleext !== 'css' && !supportedCssSupersets.includes(options.styleext!)) {
```

Comment:
Could you just add `css` to the allowed list to simplify this check?

Reply:
Done. I also changed the variable name since `css` being a superset of `css` sounds weird 👍